README.md

```diff
@@ -52,14 +53,71 @@ The library provides automatic functionality to restrict cargo drops on your pla
 
 Players will be unable to drop cargo (excluding players and construction robots) to planets with that name before researching the technology.
 
-### Support for moons
+## Support for moons
```

Comment:
This is part of the API reference. Use ### here and #### for any further subsections.